<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCRModuleWidget</class>
 <widget class="QWidget" name="PCRModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1720</width>
    <height>998</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnPosition">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>19</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>位置校准</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTemperature">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>温度校准</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLight">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>19</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>光学校准</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStackedWidget" name="moduleStackedWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>58</y>
     <width>1681</width>
     <height>916</height>
    </rect>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <zorder>moduleStackedWidget</zorder>
  <zorder>btnPosition</zorder>
  <zorder>btnTemperature</zorder>
  <zorder>btnLight</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
